### ISSUE TITLE
app crash on live streaming

### ISSUE THREAD

**Issue by martinhanes** [MarhyCZ/ivysilani_tvOS#3]
on entering the 'live streaming' section, the app crashes

**Comment by janmisar** [comment]
I've sent PR with fix. Hopefully will be merged soon and deployed on github pages 🤞 

I really don't want to host my own instance and bundle new native app with my private URL 😬

**Comment by MarhyCZ** [comment]
No problem, will do it right away O:) Thanks a lot!

**Comment by MarhyCZ** [comment]
Ok, I packaged it and published new build. Does it work now? (don't have access to aTV right now. )

**Comment by martinhanes** [comment]
Hi everyone, it works! :-)) thanks a lot for the fix!

Martin

**Comment by martinhanes** [comment]
Guys, thanks again, it's a life(stream) saver! :-) While still talking about broken LCR - isn't analogical problem with primaplay as well? At my app, the tiles don't have text until it gains focus. Isn't it analogical problem?

Thanks.

![UNADJUSTEDNONRAW_thumb_17b7](https://user-images.githubusercontent.com/13551360/64488287-72f95f80-d246-11e9-8859-ed50c6adde47.jpg)
![UNADJUSTEDNONRAW_thumb_17b6](https://user-images.githubusercontent.com/13551360/64488288-7391f600-d246-11e9-8c62-3ceae8d2d688.jpg)

**Comment by janmisar** [comment]
> Ok, I packaged it and published new build. Does it work now? (don't have access to aTV right now. )

Yeah, works great, thanks! I can finally watch 🏀 world cup now ❤️ 😄
